package/MDAnalysis/coordinates/XDR.py

```diff
@@ -41,6 +41,24 @@ def offsets_filename(filename, ending='npz'):
                                                         ending=ending))
 
 
+def read_numpy_offsets(fname):
+    """read offsets into a dictionary
+
+    Parameters
+    ----------
+    fname : str
+        filename of offsets
+
+    Returns
+    -------
+    offsets: dict
+        dict with offsets
+    """
+    with open(fname) as f:
```

Comment:
I had to fix a few occurrences of that pattern in #683 because they were 
failing. In general, I would encourage to always specify if we are 
working with bytes or with strings to avoid confusion.

On 29/01/16 23:15, kain88-de wrote:
>
> In package/MDAnalysis/coordinates/XDR.py 
> <https://github.com/MDAnalysis/mdanalysis/pull/656#discussion_r51323737>:
>
> > @@ -41,6 +41,24 @@ def offsets_filename(filename, ending='npz'):
> >                                                          ending=ending))
> >
> >
> > +def read_numpy_offsets(fname):
> > +    """read offsets into a dictionary
> > +
> > +    Parameters
> > +    ----------
> > +    fname : str
> > +        filename of offsets
> > +
> > +    Returns
> > +    -------
> > +    offsets: dict
> > +        dict with offsets
> > +    """
> > +    with open(fname) as f:
>
> only to |open| |np.load| doesn't have argument for that. So I assume 
> it will deal with it internally.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/MDAnalysis/mdanalysis/pull/656/files#r51323737>.
>